packages/expo-linear-gradient/src/NativeLinearGradient.web.tsx

```diff
@@ -32,50 +38,74 @@ export default class NativeLinearGradient extends React.PureComponent<Props, Sta
     }
   };
 
-  getAngle(): string {
-    const startPoint = this.props.startPoint ? this.props.startPoint : [0.5, 0.0];
-    const endPoint = this.props.endPoint ? this.props.endPoint : [0.5, 1.0];
+  getControlPoints = (): Point[] => {
+    const { startPoint, endPoint } = this.props;
+
+    let correctedStartPoint: Point = [0.5, 0.0];
+    if (startPoint && Array.isArray(startPoint)) {
+      correctedStartPoint = [
+        startPoint[0] ? startPoint[0] : 0.5,
```

Comment:
What if the start point coordinate is legitimately 0?
```suggestion
        startPoint[0] != null ? startPoint[0] : 0.5,
```